dev_guide/builds/build_strategies.adoc

```diff
@@ -118,15 +118,16 @@ documentation] for information on how S2I scripts are used.
 
 There are two ways to make environment variables available to the
 xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build[source build]
-process and resulting \image: xref:environment-files[environment files] and
-xref:buildconfig-environment[*BuildConfig* environment] values.
+process and resulting image: xref:environment-files[environment files] and
```

Comment:
@bparees @bmcelvee - the build system barfed on the use of the `image:` here. It thought that you were trying to incorporate an image tag :P. I had no idea that it would be a problem, but I guess we should have escaped it.

I have fixed this in the branches and the master.

Reply:
hm, well i deleted the \ because it was showing up in the rendered docs.

Reply:
How strange. It rendered correctly when I built it locally. Thanks for fixing it, @vikram-redhat!

Reply:
No worries. I should have clarified. This was failing on the portal docs, which is stricter with this stuff.